regions_www/m/pct_shiny_funs.R

```diff
@@ -0,0 +1,758 @@
+#################
+# Name, number and colour functions
+#################
+
+## Region names [NB copy of this also in pct-scripts/00_setup_and_funs - if modify here, modify there too]
+get_pretty_region_name <- function(region_name, the = T){
+  if (the == T) {
+    region_name <- gsub("isle-of-wight", "the-isle-of-wight", region_name, perl=TRUE)
+    region_name <- gsub("north-east", "the-north-east", region_name, perl=TRUE)
+    region_name <- gsub("west-midlands", "the-west-midlands", region_name, perl=TRUE)
```

Comment:
With this:

```R
 get_pretty_region_name("isle-of-wight")
```
It might be easier just to map it:

```R
regions_thes <- c(
  "isle-of-wight" = "The Isle of Wight", 
  "north-east", "The North East",
  "west-midlands", "The West Midlands"
)
if(the == T & !is.na(regions_thes[region_name])) {
  return(regions_thes[region_name])
}
```

Reply:
There's already a function that does this work.

Reply:
@Robinlovelace mind expanding on that 

> There's already a function that does this work.

Do you mean a function that creates the region names?  I can't find it, mind pointing it out?

Reply:
Will do, on way to wedding though. Look for bbmisc in pct-scripts. U should find it. Can check Monday if not.

Reply:
Ah, in a different repo, thanks.  It should be the same in both places than.